content/automate/ManageIQ/System/CommonMethods/Utils.class/__methods__/log_object.rb

```diff
@@ -18,6 +19,28 @@ def self.log_and_exit(msg, exit_code, handle = $evm)
               handle.log('info', "Script ending #{msg} code : #{exit_code}")
               exit(exit_code)
             end
+
+            # If you want to create a notification and log a message without specifying a handle using the global $evm
+            #    ManageIQ::Automate::System::CommonMethods::Utils::LogObject.log_and_notify(:level, msg, subject)
+            #
+            # If you want to create a notification and log a message using a specific handle
+            #    ManageIQ::Automate::System::CommonMethods::Utils::LogObject.log_and_notify(:level, msg, subject, handle)
+            #
+            def self.log_and_notify(level, message, subject, handle = $evm)
+              handle.create_notification(:level => notify_level(level.downcase.to_s), :message => message, :subject => subject)
+              handle.log(log_level(level.downcase.to_s), message)
+            end
+
+            def self.log_level(level)
```

Comment:
@billfitzgerald0120 What is this trying to do, does notify use warning instead of warn, if the caller is passing in level couldn't we enforce that the level be "info","warn",or"error"

Reply:
@mkanoor yes, notify uses warning and the log uses warn. The notify accepts info, warning, 
 error and success.  The log accepts info, warn and error.  For level, If a invalid value like 'fred' is passed in then it is changed to 'info' for both notify and log.